### ISSUE TITLE
API error 'Module is not connected.' appears on Settings for second admin user

### ISSUE THREAD

**Issue by wpdarren** [google/site-kit-wp#6891]
## Bug Description

While testing #6861 and noticed a console API error. `Module is not connected.` 

As the second admin user, I went to the view Site Kit settings and the error appeared.

This doesn't appear for the first admin user.

![image.png](https://images.zenhubusercontent.com/5f99965247ef5a1da6985d7d/e35afba5-4bde-488d-9ee7-670728795920)

## Steps to reproduce

1. Connect Sit Kit with Analytics.
2. Share the Analytics module via the Dashboard Sharing modal.
3. Login as a second admin and navigate to Analytics settings.
4. See error in console.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by bethanylang** [comment]
@wpdarren Is this issue still reproducible?

**Comment by wpdarren** [comment]
@bethanylang yes, this issue still exists.  The steps were out of date, so I have updated them. 

![image](https://github.com/google/site-kit-wp/assets/73545194/f2a99e50-5f20-4ff9-8e49-e95b201ac606)

**Comment by bethanylang** [comment]
@wpdarren Thanks much!

**Comment by zutigrm** [comment]
@wpdarren I am trying to go through the steps to reproduce, but I am not seeing this error, am I doing something wrong:

https://github.com/user-attachments/assets/934f2b71-930e-453f-a4f8-a0dbec848193

Are still able to reproduce this?
Thanks

**Comment by wpdarren** [comment]
@zutigrm I haven't seen this error in ages, but I just checked and no it's no longer appearing, so we can close this.
